src/test/java/com/google/api/codegen/testdata/java_test_test_library.baseline

```diff
@@ -84,8 +89,16 @@ public class LibraryServiceTest {
   @Test
   @SuppressWarnings("all")
   public void createShelfTest() {
+    Shelf response = Shelf.newBuilder()
```

Comment:
all the variables for the expected response should have "expected" in the name, otherwise it's confusing what they are.